lib/cli/init.ts

```diff
@@ -34,9 +34,14 @@ export function init(): void {
 
     let logDirectory: string = _.get(config, 'lsc.Log.Path');
     let fluentD = _.get(config, 'lsc.Log.FluentD', {});
+    let format = _.get(config, 'lsc.Log.Format', {});
 
-    global.LabShare.Logger = new Logger({
+    // default logger format - for text
+    global.LabShare.LoggerFormat = LoggerFormat;
```

Comment:
Can we avoid setting another global? Ideally, the logger instance would be accessed as a singleton.

Reply:
@KalleV  ok I will remote the Format override and define it as a static method